app/controllers/application_controller.rb

```diff
@@ -16,6 +16,14 @@ class ApplicationController < ActionController::Base
 
   rescue_from Pundit::NotAuthorizedError, with: :user_not_authorized
 
+  def after_sign_in_path_for(resource)
+    if resource.require_2sv?
```

Comment:
Now I think about it, I think it'd be better if this only triggered if 2SV is required and they haven't got it already set up. That way, if we flag someone as needing 2SV and they get it turned off at a later date (in an emergency, through a bug, as part of some future feature), we don't lose the information that this user is flagged as requiring 2SV.

So rather than changing the flag, just ignore it if 2SV is configured. This also protects against weird data failures or bugs where the 2SV flag could be turned back on once they have 2SV set up, leaving them stuck in a loop.